lang/cs/Org.Apache.REEF.Driver/Bridge/ClrSystemHandlerWrapper.cs

```diff
@@ -256,6 +252,23 @@ public static void Call_ClrSystemDriverRestart_OnNext(ulong handle)
             }   
         }
 
+        public static ulong[] Call_ClrSystemRestartHandler_OnRestart(
```

Comment:
Do these methods have to be public, or can we get away with `internal`, as they are only called from C++ code in the same DLL?